### PR TITLE
bugfix(unique_selector): don't include if null

### PR DESCRIPTION
**Pull request by dabrady**
*Contributes to [DG-138](https://prodperfect.atlassian.net/browse/DG-138)*

We're seeing errors downstream in code that is not expecting the `all_attrs` object of our event payloads to contain null values, which is possible with the `unique_selector` field I added in #69. This hotfix PR stops the bleeding by ensuring we don't include it when it doesn't have a value; further work is required to shore-up the consumption code against null values in the long run.

I'm going to fast-track this fix through Tier 1 & Tier 2 to make the screaming stop.

fyi @rhymiz @pevner-p2

### CODE REVIEW

File: lib/helpers/getBasicDomNodeProfile.js [hunk 1]
```diff
@@ -14,24 +14,29 @@ function getAttr(el, name) {
...
+  /**
+   * This is a short, unique CSS selector for quickly grabbing this node from the DOM.
+   * It serves a different purpose than a full node 'path', which potentially holds
+   * much more information about the DOM structure.
+   *
+   * NOTE(dabrady) We are putting this in the `all_attrs` blob out of a variety of
+   * concerns, but primarily because it will let us evalutate our use-case for this
+   * data faster and with minimal effort.
+   */
+  const unique_selector = uniqueSelector(el);
```
[the remark this answers is not in the document]
Comment (dabrady) [reply lib/helpers/getBasicDomNodeProfile.js:28]: 😒

File: lib/helpers/getBasicDomNodeProfile.js [hunk 2]
```diff
@@ -14,24 +14,29 @@ function getAttr(el, name) {
 }
 
 export default function getBasicDomNodeProfile(el) {
+  let all_attrs = getDomNodeAttributes(el, EXCLUDE_VALUE_REGEX);
```
Comment (pevner-p2) [inline lib/helpers/getBasicDomNodeProfile.js:17]: Maybe this is a dumb question, but does this need to be a copy, rather than the original?

**Review by pevner-p2 — APPROVED** [review]
As far as I understand this code it looks good to me.

File: lib/helpers/getBasicDomNodeProfile.js (hunk 2 above)
Comment (dabrady) [reply lib/helpers/getBasicDomNodeProfile.js]: Good question: `getDomNodeAttributes` is computing a literal value, not accessing & returning a reference to something, it shouldn't matter here 👍🏻

Objects in JS are indeed mutable, but we're not closing over anything in this case: `getDomNodeAttributes` returns an object literal, and then we're just shoving it into another object literal, and then serializing it later on. They're all intermediate objects, and nothing is maintaining a reference to them: they'll fall out of scope each time the function call frame is popped off the stack.

**Comment by dabrady** [comment]
Associated CI build is here: https://semaphoreci.com/prodperfect/prodperfect-keen-tracking-js/branches/master/builds/1544

**Review by rhymiz — APPROVED** [review]

**Review by rhymiz — APPROVED** [review]

**Pull request merged**